### PR TITLE
docs: added community contribution mini-guide

### PR DESCRIPTION
**Pull request by igpetrov**
Add a community contribution guide.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +61,20 @@ Connector runtimes provided by the Community:
...
 | OpenAI Moderation | - | [GitHub](https://github.com/MaxTru/camunda-connector-openai-moderation) | [README](https://github.com/MaxTru/camunda-connector-openai-moderation/blob/main/README.md) | MIT       |         [![](https://img.shields.io/badge/Lifecycle-Proof%20of%20Concept-blueviolet)](https://github.com/Camunda-Community-Hub/community/blob/main/extension-lifecycle.md#proof-of-concept-)  |
 
+# Adding connectors
+
+You are able to add your connector in just a few steps:
+- Be inspired with a brilliant idea!
+- [Implement it](https://docs.camunda.io/docs/components/connectors/custom-built-connectors/connector-sdk/#creating-a-custom-connector).
+- Test it with a [Self-Managed Camunda Platform](https://docs.camunda.io/docs/self-managed/about-self-managed/) instance.
+- Publish it at GitHub.
```
Comment (bastiankoerber) [inline README.md:70]: as own repository?

File: README.md [hunk 2]
```diff
@@ -60,6 +61,20 @@ Connector runtimes provided by the Community:
 |  GitHub Dispatch          |            -           |      [GitHub](https://github.com/camunda-community-hub/camunda-8-connector-github-dispatch)           |   [README](https://github.com/camunda-community-hub/camunda-8-connector-github-dispatch/blob/main/README.md)            |   Apache 2.0      |         [![](https://img.shields.io/badge/Lifecycle-Proof%20of%20Concept-blueviolet)](https://github.com/Camunda-Community-Hub/community/blob/main/extension-lifecycle.md#proof-of-concept-)  |
 | OpenAI Moderation | - | [GitHub](https://github.com/MaxTru/camunda-connector-openai-moderation) | [README](https://github.com/MaxTru/camunda-connector-openai-moderation/blob/main/README.md) | MIT       |         [![](https://img.shields.io/badge/Lifecycle-Proof%20of%20Concept-blueviolet)](https://github.com/Camunda-Community-Hub/community/blob/main/extension-lifecycle.md#proof-of-concept-)  |
 
+# Adding connectors
+
+You are able to add your connector in just a few steps:
+- Be inspired with a brilliant idea!
```
Comment (bastiankoerber) [inline README.md:67]: maybe we can an option to only share the idea. Sth. Like this:
You want to develop it yourself and share it with the community?
Or do you just want to share your idea with the community? Link to feature request****

**Review by bastiankoerber — COMMENTED** [review]
in general I like it. Before you merge it, can you please add the issue template to it?

File: README.md (hunk 1 above)
Comment (igpetrov) [reply README.md:70]: yes. We need to review the code first and make sure that at least essentials are present: readme, license, etc. Not all users have access to the `camunda-community-hub` in the beginning. We can transfer it later with the ticket if that is required.

File: README.md (hunk 2 above)
Comment (igpetrov) [reply README.md]: Added a side note on a feature request.

**Comment by igpetrov** [comment]
Hi @bastiankoerber,

Please see updated.

**Review by bastiankoerber — APPROVED** [review]
looks good, thank you Igor :-)

**Review by markfarkas-camunda — APPROVED** [review]

**Pull request merged**